code-studio/src/css/plc/teacher_dashboard.scss

```diff
@@ -24,3 +24,47 @@
   }
 }
 
+.course_unit_sections {
+  display: flex;
```

Comment:
this might be the first time we're using flexbox.  we should be careful because apparently IE11 is only considered "partially compatible" due to a lot of bugs: http://caniuse.com/#feat=flexbox  Does this work okay on all our browsers?